artifactory/services/releasebundle.go

```diff
@@ -76,3 +83,26 @@ func (rs *releaseService) ImportReleaseBundle(filePath string) (err error) {
 	log.Info("Release Bundle Imported Successfully")
 	return
 }
+
+func (rs *releaseService) IsReleaseBundleExistInRbV2(project, bundleNameAndVersion string) (bool, error) {
+	httpClientsDetails := rs.ArtDetails.CreateHttpClientDetails()
+	project = "project=" + project
+
+	rtUrl := strings.Replace(rs.ArtDetails.GetUrl(), "/artifactory", "", 1)
```

Comment:
Once we transition to the V2 code, we should no longer need to manually remove the Artifactory endpoint; we can simply append the new endpoint.